### ISSUE TITLE
Message log. Color contrast is insufficient

### ISSUE THREAD

**Issue by amazingphilippe** [cds-snc/notification-planning#68]
### Description of issue
Content under "Message log" is only AA compliant. We can make it AAA compliant.

![image](https://user-images.githubusercontent.com/5230720/136052178-598697f6-6f42-4104-a693-9b42cb0aa741.png)

### Potential fix

Using black as the text color would work pretty good. 

### Resources
https://color.review/check/6F777B-FFFFFF

**Comment by amazingphilippe** [comment]
This is fixed, as far as I can tell.
